docs/timeline/index.html

```diff
@@ -461,6 +461,18 @@ <h3 id="groups">Groups</h3>
       <td>no</td>
       <td>Provides a means to toggle the whether a group is displayed or not. Defaults to <code>true</code>.</td>
     </tr>
+    <tr>
+      <td>nestedGroups</td>
```

Comment:
Now that I look at it I think "subGroups" would have been the better work, but I'm ok with this.